scripts/complete_liquidity_provision.js

```diff
@@ -64,12 +65,27 @@ module.exports = async callback => {
 
     assert(argv.fleetSize % 2 == 0, "Fleet size must be a even number for easy deployment script")
 
-    console.log("1. Check for sufficient funds")
+    console.log("1. Preponded checks")
```

Comment:
I propose "Sanity checks". If you prefer the current version, typo: prepon**~d~**ed.